src/Magnum/GL/Test/ShaderGLTest.cpp

```diff
@@ -40,6 +42,18 @@
 
 namespace Magnum { namespace GL { namespace Test { namespace {
 
+constexpr GL::Version shaderCompileGLVersion =
+#ifndef MAGNUM_TARGET_GLES
+    #ifndef CORRADE_TARGET_APPLE
+    Version::GL210
+    #else
+    Version::GL310
+    #endif
+    ;
+#else
+Version::GLES200;
+#endif
```

Comment:
Should I just move this back into each test case? Because code re-use is pretty minimal across what I've seen in the tests

Reply:
Yes, please, that makes them less prone to breakages if certain cases would need a different version. The tests are mostly write-only anyway, so the duplication isn't that much of a problem.